incident/models/incident_index_page.py

```diff
@@ -255,7 +255,7 @@ def serve(self, request, *args, **kwargs):
         # simultaneously
         response['Cache-Tag'] = self.get_cache_tag()
 
-        if request.is_ajax():
+        if request.headers.get('x-requested-with') == 'XMLHttpRequest':
```

Comment:
Same as above. If we remove the conditional in `get_context`, we don't need it here, either.